.evergreen/run-mod-wsgi-tests.sh

```diff
@@ -19,7 +19,9 @@ fi
 PYTHON_VERSION=$(${PYTHON_BINARY} -c "import sys; sys.stdout.write('.'.join(str(val) for val in sys.version_info[:2]))")
 
 # Ensure the C extensions are installed.
-${PYTHON_BINARY} -m pip install -e .
+${PYTHON_BINARY} -m venv .venv
+source .venv/bin/activate
+python -m pip install -e .
```

Comment:
I suspect we need to use --system-site-packages so that the toolchain installed version of mod_wsgi  can be used.

Reply:
Done